packages/smooth_app/lib/pages/product/product_image_server_button.dart

```diff
@@ -37,50 +37,47 @@ class ProductImageServerButton extends StatelessWidget {
     if (!loggedIn) {
       return;
     }
+
+    List<int>? result;
     if (context.mounted) {
```

Comment:
`if (!context.mounted)` is still not working?

Reply:
The way I've changed the code is the only way to remove all warnings.

Before the code was a bit strange, with multiple if without any bodies.
I have simply moved the code within and the Linter seems happy with it